### PR TITLE
feature/UT-1294: inscription destination address implemented

### PR DESCRIPTION
**Pull request by gpg4b**

### CODE REVIEW

File: browser-extension/src/background/api.ts [hunk 1]
```diff
@@ -665,13 +665,10 @@ class Api {
...
-    if (this.wallet[type].typeAddress === 1) {
-      this.wallet[type].key = this.wallet.root.key.Derive(this.path(type), for_script);
-      this.wallet[type].address = this.wallet[type].key.GetP2WPKHAddress(this.network);
-      return true;
-    }
     this.wallet[type].key = this.wallet.root.key.Derive(this.path(type), for_script);
-    this.wallet[type].address = this.wallet[type].key.GetP2TRAddress(this.network);
+    this.wallet[type].address = this.wallet[type].typeAddress === 1
+        ? this.wallet[type].key.GetP2WPKHAddress(this.network)
+        : this.wallet[type].key.GetP2TRAddress(this.network);
```
Comment (l2xl) [inline browser-extension/src/background/api.ts:671]: А как деривируется сам ключ? Всмысле, там не только кодировка адреса разная, но и путь деривации должен меняться. Это строчка 668, но по ней не видно, как меняется путь при изменении настроек

Comment (dipspb) [reply browser-extension/src/background/api.ts:671]: Before my changes it has the same derivation for p2tr and p2wpkh. So I leave it as it was implemented.

Comment (gpg4b) [reply browser-extension/src/background/api.ts:671]: Before my changes it has the same derivation for p2tr and p2wpkh. So I leave it as it was implemented.

**Review by DevOraTor18 — APPROVED** [review]

File: browser-extension/src/background/api.ts (hunk 1 above)
Comment (dipspb) [reply browser-extension/src/background/api.ts:671]: @DevOraTor18 I suppose question has been addressed to you.

Comment (gpg4b) [reply browser-extension/src/background/api.ts:671]: @DevOraTor18 I suppose question has been addressed to you.

Comment (DevOraTor18) [reply browser-extension/src/background/api.ts:671]: @l2xl the path to obtain the address is formed in the path method

**Pull request merged**